embedded-database-spring-test/src/test/java/io/zonky/test/db/LiquibaseRefreshIntegrationTest.java

```diff
@@ -100,9 +101,9 @@ public void afterTestClass(TestContext testContext) {
         DatabaseContext databaseContext = applicationContext.getBean(DatabaseContext.class);
         DatabaseProvider databaseProvider = applicationContext.getBean("dockerPostgresDatabaseProvider", DatabaseProvider.class);
 
-        verify(databaseContext, times(4)).reset();
+        verify(databaseContext, atLeast(2)).reset();
```

Comment:
This change could significantly affect the library's performance, so it's necessary to carefully consider and justify every such change.